docs/source/getting_started/lightly_at_a_glance.rst

```diff
@@ -94,28 +105,55 @@ with the normalized temperature-scaled cross entropy loss and simple stochastic
           `colab playground <https://colab.research.google.com/drive/1ubepXnpANiWOSmq80e-mqAxjLx53m-zu?usp=sharing>`_.
 
 
-Put everything together in an embedding model and train it for 10 epochs on a single GPU.
+Train the model for 10 epochs.
 
 .. code-block:: python
+    
+    for epoch in range(10):
+        for (x0, x1), _, _ in dataloader:
+            
+            x0 = x0.to(device)
+            x1 = x1.to(device)
 
-    import lightly.embedding as embedding
+            z0 = model(x0)
+            z1 = model(x1)
 
-    # put all the pieces together in a single pytorch_lightning trainable!
-    embedding_model = embedding.SelfSupervisedEmbedding(
-        model,
-        criterion,
-        optimizer,
-        dataloader)
+            loss = criterion(z0, z1)
+            loss.backward()
+
+            optimizer.step()
+            optimizer.zero_grad()
 
-    # do self-supervised learning for 10 epochs
-    embedding_model.train_embedding(gpus=1, max_epochs=10)
 
 Congrats, you just trained your first model using self-supervised learning!
 
-You can also train the model using PyTorch Lightning directly.
+You can of course also use `PyTorch Lightning <https://www.pytorchlightning.ai/>`_ to implement and train your model.
 
 .. code-block:: python
 
+    class SimCLR(pl.LightningModule):
```

Comment:
can you add the `pytorch-lightning` import here?

Reply:
Done